Add confirmation page for workshops (#2367)
pulumi/docs: content/confirmation/workshop.md

---
title: Thank You | Workshop

> meta_desc: This page is provides confirmation about register for a Pulumi workshop.

Reviewer: ```suggestion
meta_desc: This page provides confirmation after registering for a Pulumi workshop.
```

Author: Good catch, thanks!

type: page
layout: confirmation

> url: /confirmation/workshop

Reviewer: Is the `url` parameter necessary? I think we might be able to delete this line.

```suggestion
```

Author: I guess it's not. I removed it.
